### PR TITLE
docs: update clusters+quickstart

### PR DESCRIPTION
**Pull request by Itxaka**
Fixes: #318 
Fixes: #453 

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

**Comment by Itxaka** [comment]
cc @nunix

**Comment by Itxaka** [comment]
Quickstart:

image

**Comment by Itxaka** [comment]
Cluster page:
Screenshot 2022-10-25 at 14-51-25 Cluster reference - Elemental - Immutable Linux for Rancher

**Comment by Itxaka** [comment]
quick start:

<details>
<summary>Image</summary>

Screenshot 2022-10-25 at 14-52-09 Quickstart - Elemental - Immutable Linux for Rancher

</details>

File: mkdocs.yml [hunk 1]
```diff
@@ -86,5 +86,6 @@ nav:
     - elementaloperatorchart-reference.md
     - kubernetesversions.md
     - smbios.md
+    - cluster-reference.md
```
Comment (fgiudici) [inline mkdocs.yml:89]: I would move up the cluster-reference, just after the machineinventoryselectortemplate-reference.md

**Review by fgiudici — APPROVED** [review]
Great job the new cluster-reference section ❤️ !

File: mkdocs.yml (hunk 1 above)
Comment (Itxaka) [reply mkdocs.yml:89]: Good point, I forgot this order is the actual column order in the page!

**Pull request merged**